chatmeleon-server/routes/user-route.ts

```diff
@@ -0,0 +1,6 @@
+import userController from "../controllers/user-controller.js"
+import { Router } from 'express';
+
+export const userRoute = Router();
+userRoute.get('/users/:id', userController.getUserById);
```

Comment:
There should be some separation lines between different methods GET, POST, PUT, PATCH, and DELETE for clarity